WordPress/wordpress-playground
# WordPress PR Previewer: Simplify the Blueprint

Replaces a series of custom steps meant to unzip and configure WordPress with passing the build zip as a preferredVersion. Here's what the updated Blueprint looks like:

```js
{
	$schema:
		'https://playground.wordpress.net/blueprint-schema.json',
	landingPage: urlParams.get('url') || '/wp-admin',
	login: true,
	preferredVersions: {
		php: "7.4",
		wp: zipArtifactUrl
	}
};
```

## Testing instructions

* Go to the WordPress PR previewer locally at http://localhost:5400/website-server/wordpress.html
* Try previewing one of the recent PRs. Confirm the installed WordPress versions is indeed not the latest stable.
